app/assets/javascripts/application.js

```diff
@@ -5,13 +5,13 @@
 //= require govuk-admin-template
 //= require admin_legacy
 
-//= require components/miller-columns
-//= require components/autocomplete
+//= require admin/components/miller-columns
+//= require admin/components/autocomplete
```

Comment:
I imagine we're not distinguishing components as being admin or not - so I'd probably suggest leaving them under components

Reply:
Components will never be used within the `frontend` side of things as we have no plans to move any of the `frontend` code to the GOVUK design system. The plan is once we remove all of the frontend code, we will be moving the whole `admin` directory up one level.

Reply:
As I understand the frontend is already moved to the design system and makes use of components.

Even if that wasn't the case I just suspect that we're going to get caught in a muddle with the component guide expecting a configuration in some places. So I feel like trying to make them distinct with an admin prefix is likely to cause some problems or inconsistencies.